src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/configs/QueueActionConfig.java

```diff
@@ -36,7 +36,7 @@
  *       "total-step-count": 20,
```

Comment:
This is confusing. I notice you have changed to use actionConfig. In that case this should be outside queue-settings block right?

Reply:
this is the step count to divide the range of queue size into steps. We can specify different step count in cache if we want to have different granularity